ut/abnormal_column_names_test.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <clickhouse/client.h>
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <tuple>
+#include <vector>
+
+class AbnormalColumnNamesTest : public testing::TestWithParam<
```

Comment:
Please rename the test to something with `Client`, it is a convention that allows easy filtering out local tests vs tests that do requests to the server.